crates/solidity/inputs/language/src/definition.rs

```diff
@@ -4074,6 +4075,118 @@ codegen_language_macros::compile!(Language(
                             name = YulIdentifier,
                             definitions = [TokenDefinition(scanner = Fragment(RawIdentifier))]
                         ),
+                        Enum(
+                            name = YulEvmBuiltin,
```

Comment:
nit: adding `Function`, also using `In` as "built-in" is two words:

```suggestion
                            name = YulEvmBuiltInFunction,
```